machine/models/seq2seq.py

```diff
@@ -8,9 +9,13 @@ class Seq2seq(BaseModel):
     and decoder.
     """
 
-    def __init__(self, encoder, decoder, decode_function=F.log_softmax):
+    def __init__(self, encoder, decoder, decode_function=F.log_softmax,
+                 uniform_init=0, glorot_init=False):
```

Comment:
Putting all this in the init like this perhaps doesn't generalise very if you want to add also other kind of initialisations. Perhaps instead we could pass a function for initialisation? Does that make sense?

Reply:
Ah I see what you mean, so we would move this initialization function outside of the model class and maybe into a util? And one would have the option of passing an initialization function to any class and have the weights of that class initialized accordingly?

Reply:
I didn't think about it that way initially but I think that makes more sense actually. So ignore this PR for now haha

Reply:
Ok, thanks!